### PR TITLE
Audit sink CRUD

### PR DESCRIPTION
**Pull request by arxeiss**
# Checklist

- [x] `make test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows [commit guidelines](./doc/guides/commit-message.md#commit-message-guidelines)

### CODE REVIEW

File: .github/workflows/changelog.yaml [hunk 1]
```diff
@@ -6,9 +6,6 @@ on:
         branches:
             - master
 
-env:
-    BIGQUERY_TABLE: jarvis-dev-268314.deployment_analytics.deployment_events
```
Comment (bakkelim) [inline .github/workflows/changelog.yaml:10]: I don't know what this is.

File: GNUmakefile [hunk 2]
```diff
@@ -24,13 +24,15 @@ fieldalignment:
 	@echo "==> Analyzer structs and rearranged to use less memory with fieldalignment..."
 	fieldalignment -fix -test=false ./...
 
-download:
-	@echo Download go.mod dependencies
-	@go mod download
-
-install-tools: download
-	@echo Installing tools from tools.go
-	@go install $$(go list -f '{{range .Imports}}{{.}} {{end}}' tools.go)
```
Comment (bakkelim) [inline GNUmakefile:33]: This command not working? Since you've changed it to hard coded installs

**Review by bakkelim — APPROVED** [review]
LGTM, just a few questions

File: .github/workflows/changelog.yaml (hunk 1 above)
Comment (arxeiss) [reply .github/workflows/changelog.yaml:10]: Me neither, but we don't use it anymore. It was removed from other places too

File: GNUmakefile (hunk 2 above)
Comment (arxeiss) [reply GNUmakefile:33]: Not works in Go 1.21 anymore. See jarvis-proto, there is it changed already too

**Pull request merged**